CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Enhanced the caching implementation of MAuth Authenticator Apachehttp
```

Comment:
Can you improve the wording here? "Enhanced" is vague, we're fixing a bug around caching.

Reply:
updated in https://github.com/mdsol/mauth-jvm-clients/pull/142/commits/dd0570b356fba0741430abc554e7265a8a7a554b